### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by ricardomcrae** [rohanb10/snoozz-tab-snoozing#94]
## Description
The name "Today Night" should read "Tonight".

## Your Device
* Browser and version: Chrome Version 98.0.4758.102 
* OS: OSX 12.2.1 

## Screenshots / Videos / Logs
![Screen Shot 2022-02-24 at 9 39 52 AM](https://user-images.githubusercontent.com/100364169/155554823-f1572f7d-e894-470d-9b6b-997ec180a840.png)

**Comment by rohanb10** [comment]
Yeah there was a very technical reason behind calling it "Tomorrow Night" instead of "Tonight".

As I was writing down the explanation behind this, I realised how dumb this reason was (from a user's perspective).

I have made this change in https://github.com/rohanb10/snoozz-tab-snoozing/commit/d44c3ebe5bad3b0ce83f6ec7dfc640c2d0047232 . Will be out with the next release

**Comment by ricardomcrae** [comment]
rohanb10 - Thank you.. I am happy to contribute to making things better for everyone. 
Thanks for actually considering it.
